src/chapters/02_docker/03_running-containers/index.md

```diff
@@ -124,6 +139,8 @@ Once the database is up and running, the WordPress application container can be
 docker run /
   --name my-wordpress /
   -e WORDPRESS_DB_HOST=wordpress-db /
+  -e WORDPRESS_DB_USER=root /
+  -e WORDPRESS_DB_NAME=wordpress /
```

Comment:
this should be enough to create a database called 'wordpress' and should make your addition above redundant